bouncer/values-live.yaml

```diff
@@ -11,9 +11,8 @@ podCommand:
 ingress:
   enabled: true
   className: "traefik"
-  annotations: {}
-    # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
+  annotations:
+    traefik.ingress.kubernetes.io/router.middlewares: allowCORS@file
```

Comment:
Here you made changes in the values-live file, but in others in values-dev.

Reply:
Nop. Thx

Reply:
Fixed